egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# This script is based on swbd/s5c/local/nnet3/run_tdnn.sh
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
+# call multi-splice.
```

Comment:
please have a look at some other chain/nnet3 scripts and add the performance indicators (dir_info and wers)

Reply:
Thank you, I have generated and added corresponding performance indicators, please check them.